VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -238,18 +245,44 @@ def post_to_wireserver(self, data):
 
                     http_util.connection.close()
                     if result.status != httplib.OK and result.status != httplib.ACCEPTED:
-                        raise Exception("encryption settings update request was not accepted")
+                        raise Exception("Encryption settings update request was not accepted")
                     return
                 else:
-                    raise Exception("no response from encryption settings update request")
+                    raise Exception("No response from encryption settings update request")
             except Exception as e:
                 retry_count += 1
                 self.logger.log("Encountered exception while posting encryption settings to Wire Server (attempt #{0}):\n{1}".format(str(retry_count), str(e)))
                 if retry_count < retry_count_max:
-                    time.sleep(5) # sleep for 5 seconds before retrying.
+                    time.sleep(5)  # sleep for 5 seconds before retrying.
                 else:
                     raise e
 
+    def post_to_wireserver(self, data):
+        """ Request EnableEncryption operation on settings file via wire server """
+        self.write_settings_file(data)
+        if not os.path.isfile(self.get_settings_file_path()):
+            raise Exception(
+                'Disk encryption settings file not found: ' + self.get_settings_file_path())
+
+        http_util = self.get_http_util()
+
+        # V3 message content
```

Comment:
New version of EncryptionSettingsUtil.py in line 36-37 references 4.0 and 3.0, this post code references a wire server version 3 and 2, is there an off-by-one somewhere?